src/main/java/org/broadinstitute/consent/http/resources/VoteResource.java

```diff
@@ -0,0 +1,102 @@
+package org.broadinstitute.consent.http.resources;
+
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import io.dropwizard.auth.Auth;
+import org.broadinstitute.consent.http.enumeration.ElectionStatus;
```

Comment:
A couple of these imports aren't being used, namely ElectionStatus, Election, and Collectors